packages/vm/tests/api/runBlockchain.spec.ts

```diff
@@ -48,7 +47,7 @@ tape('runBlockchain', (t) => {
   t.test('should run with genesis block', async (st) => {
     try {
       const common = new Common({ chain: 'goerli', hardfork: 'chainstart' })
-      const genesis = createGenesis({ common })
+      const genesis = Block.genesis(undefined, { common })
```

Comment:
Why this `undefined` here instead of `{}`?

Reply:
Both work since the param defaults to `{}` if undefined.